tests/lib/rules/yoda.js

```diff
@@ -159,7 +186,18 @@ ruleTester.run("yoda", rule, {
         { code: "if (x < `2` && x !== `-3`) {}", options: ["always", { onlyEquality: true }], parserOptions: { ecmaVersion: 2015 } }
     ],
     invalid: [
-
+        {
+            code: "if (x <= 'bar' || 'foo' < x) {}",
```

Comment:
this is invalid because `"foo"` is greater than `"bar"` 

![image](https://user-images.githubusercontent.com/26347874/77098180-f1c22700-6a37-11ea-881b-6b1a078e8152.png)

Reply:
This is a valid range test. It checks if `x` is not between `bar` and `foo`.

Reply:
@mdjermanovic 

I think it's in invalid because `(x <= 'bar' || 'foo' < x)` is `yoda` and `always` expects only `yoda` but `exceptRange` with `always` expects `non yoda` conditions. 

right !

Reply:
fixed